keras/layers/noise.py

```diff
@@ -90,3 +90,64 @@ def get_config(self):
         config = {'rate': self.rate}
         base_config = super(GaussianDropout, self).get_config()
         return dict(list(base_config.items()) + list(config.items()))
+
+
+class AlphaDropout(Layer):
```

Comment:
There was `noise_shape` when it was in Dropout/core layers. I'll just add it back, I guess